src/tokens/backgroundColor.js

```diff
@@ -39,7 +39,14 @@ const light = core => ({
   RadioDot: core.Color.Neutral100,
   RadioDotChecked: core.Color.Primary300,
   RadioDotDisabled: core.Color.Neutral200,
-  RadioDotDisabledChecked: core.Color.Neutral500
+  RadioDotDisabledChecked: core.Color.Neutral500,
+  DropdownLabelDefault: core.Color.NeutralWhite,
+  DropdownLabelSecondary: core.Color.Neutral100,
```

Comment:
Like we discussed in chat, this can probably just use from the input unless you think this could have different values eventually?

Reply:
I misunderstood. I thought you didn't want that. I'll remove them. If we need dropdown specific ones we can add them in at that time.

Reply:
We do need a backgroundColor token for our menu dropdowns, like the Actions menu and the Usermenu.